lib/matrix_api_lite/generated/api.dart

```diff
@@ -4648,7 +4693,7 @@ class Api {
     if (response.statusCode != 200) unexpectedResponse(response, responseBody);
     final responseString = utf8.decode(responseBody);
     final json = jsonDecode(responseString);
-    return ignore(json);
+    return json as Map<String, Object?>;
```

Comment:
this is weird, I can't find that in the spec text.

Reply:
The spec now returns an event_id like this:

```json
{
  "event_id": "$YUwRidLecu:example.com"
}
```

so returning json is at least not wrong but fetching the string, like we do with the sendMessage method would be better